i18n/uk.toml

```
@@ -0,0 +1,247 @@
+# i18n strings for the Ukrainian (main) site.
+
+[caution]
+# other = "Caution:"
+other = "Увага:"
+
+[cleanup_heading]
+# other = "Cleaning up"
+other = "Очистка"
+
+[community_events_calendar]
+# other = "Events Calendar"
+other = "Календар подій"
+
+[community_forum_name]
+# other = "Forum"
+other = "Форум"
+
+[community_github_name]
+other = "GitHub"
+
+[community_slack_name]
+other = "Slack"
+
+[community_stack_overflow_name]
+other = "Stack Overflow"
+
+[community_twitter_name]
+other = "Twitter"
+
+[community_youtube_name]
+other = "YouTube"
+
+[deprecation_warning]
+# other = " documentation is no longer actively maintained. The version you are currently viewing is a static snapshot. For up-to-date documentation, see the "
+other = " Документація більше не підтримується. Версія, яку ви зараз переглядаєте, є статичною. Для перегляду актуальної документації дивіться "
+
+[deprecation_file_warning]
+# other = "Deprecated"
+other = "Застаріла версія"
+
+[docs_label_browse]
+# other = "Browse Docs"
+other = "Переглянути документацію"
+
+[docs_label_contributors]
+# other = "Contributors"
+other = "Контриб'ютори"
+
+[docs_label_i_am]
+# other = "I AM..."
+other = "Я..."
+
+[docs_label_users]
+# other = "Users"
+other = "Користувачі"
+
+[feedback_heading]
+# other = "Feedback"
+other = "Ваша думка"
+
+[feedback_no]
+# other = "No"
+other = "Ні"
+
+[feedback_question]
+# other = "Was this page helpful?"
+other = "Чи була ця сторінка корисною?"
+
+[feedback_yes]
+# other = "Yes"
+other = "Так"
+
+[latest_version]
+# other = "latest version."
+other = "остання версія."
+
+[layouts_blog_pager_prev]
+# other = "<< Prev"
+other = "<< Назад"
+
+[layouts_blog_pager_next]
+# other = "Next >>"
+other = "Вперед >>"
+
+[layouts_case_studies_list_tell]
+# other = "Tell your story"
+other = "Розкажіть свою історію"
+
+[layouts_docs_glossary_aka]
+# other = "Also known as"
+other = "Також відомий як"
+
+[layouts_docs_glossary_description]
+# other = "This glossary is intended to be a comprehensive, standardized list of Kubernetes terminology. It includes technical terms that are specific to Kubernetes, as well as more general terms that provide useful context."
+other = "Даний словник створений як повний стандартизований список термінології Kubernetes. Він включає в себе технічні терміни, специфічні для Kubernetes, а також більш загальні терміни, необхідні для кращого розуміння контексту."
+
+[layouts_docs_glossary_deselect_all]
+# other = "Deselect all"
+other = "Відмінити вибір усіх тегів"
```

А звідки взялось "тегів"?
```suggestion
other = "Очистити вибір"
```

---

це стосується тегів словника, тому це здалось мені логічним